src/bap/bir.py

```diff
@@ -253,7 +253,9 @@ class Values(Map) :
     It is a mapping from the tid of a preceeding block,
     to an expression that denotes a value.
     """
-    pass
+    def __init__(self, *args):
+        super(Map,self).__init__(args)
```

Comment:
pylint doesn't like it, for some reason it believes that the first parameter of `super` should _always_ be the current class. I think we can try to hush it with `# pylint: disable=bad-super-call` added to this line. Alternatively, we can just call the `ADT` constructor directly, e.g., `ADT.__init__(args)`

Reply:
sometimes I just ignore pylint's warning like this, lol. Maybe it's not a very good habit.